### ISSUE TITLE
I have now integrated the drop-in dependency library, can I make paypal a separate button for the user to operate?

### ISSUE THREAD

**Issue by Yongyong-Zhou** [braintree/braintree-android-drop-in#327]
Excuse me, I want to ask a question:
I have integrated the implementation of 'com.braintreepayments.api:drop-in:6.0.1', but we need to make paypal a separate button for users to operate, now only after clicking the payment button in braintree-android-drop-in -Select Paypal in the pop-up window, can you tell me the steps to call paypal directly? Or is there a direct call to paypal demo?

If I want to do this function, I have already referenced the drop-in dependent library, do I need to introduce the Paypal dependent library separately?

Thank you

**Comment by Yongyong-Zhou** [comment]
![image](https://user-images.githubusercontent.com/86334406/150294163-f407eb8b-c56e-4aa9-a09c-6ea893679438.png)
This popup is called by DropInRequest dropInRequest = new DropInRequest();, can I call the Paypal function myself?

**Comment by sarahkoop** [comment]
Closing as duplicate of #328
